apps/storefront/pages/god-praksis/tilgjengelighet/kontrast.mdx

```diff
@@ -0,0 +1,181 @@
+
+<Card color='third'>
+  <Card.Content>
+    <List.Root>
+      <List.Heading
+        level={3}
+        size='xsmall'
+      >
+        Gjeldende regelverk, WCAG 2.1
+      </List.Heading>
+      <List.Unordered>
+        <List.Item>
+          **1.4.3 Kontrast (minimum) (Nivå AA)**: Kontrastforholdet mellom
+          teksten og bakgrunnen er minst 4,5:1. [Retningslinjer for universell
+          utforming av nettinnhold (WCAG) 2.1
+          (w3.org)](https://www.w3.org/Translations/WCAG21-no/#contrast-minimum)
+        </List.Item>
+        <List.Item>
+          **1.4.11 Kontrast for ikke-tekstlig innhold (Nivå AA)**: Den visuelle
+          presentasjonen av det følgende har et kontrastforhold på minst 3:1 mot
+          farge(r) som ligger ved siden av. [Retningslinjer for universell
+          utforming av nettinnhold (WCAG) 2.1
+          (w3.org)](https://www.w3.org/Translations/WCAG21-no/#non-text-contrast)
+        </List.Item>
+      </List.Unordered>
+    </List.Root>
+  </Card.Content>
+</Card>
+
+<br />
+
+<Card color='second'>
+  <Card.Content>
+    <List.Root>
+      <List.Heading
+        level={3}
+        size='xsmall'
+      >
+        Fremtidig eller strengere:
+      </List.Heading>
+      <List.Unordered>
+        <List.Item>
+          **1.4.6 Kontrast** (forbedret) (Nivå AAA): Den visuelle presentasjonen
+          av tekst og bilder av tekst har et kontrastforhold på minst 7:1,
+          unntatt uvesentlig tekst og skriftstørrelser større enn 18px eller
+          14px fet. [Retningslinjer for universell utforming av nettinnhold
+          (WCAG) 2.1
+          (w3.org)](https://www.w3.org/Translations/WCAG21-no/#contrast-enhanced)
+        </List.Item>
+        <List.Item>
+          **WCAG 2.2: 2.4.13** Focus Appearance (Nivå AAA), om utseende til
+          fokusmarkering krever at fokusindikator har en kontrastverdi på 3:1
+          mellom samme piksler i fokusert og ikke-fokusert tilstand.
+          [Understanding Success Criterion 2.4.13: Focus Appearance | WAI |
+          W3C](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
+        </List.Item>
+        <List.Item>
+          **WCAG 3** har et krav om farge og kontrast, visuell kontrast i tekst
+          (sølv): Sørg for tilstrekkelig kontrast mellom tekst i forgrunnen og
+          bakgrunnen for teksten. Her brukes det en ny metode, med navn APCA,
+          for å regne ut kontrasten.
+        </List.Item>
+      </List.Unordered>
+    </List.Root>
+  </Card.Content>
+</Card>
+
+## Ny metode utforskes
+
+En ny metode som er mer presis enn dagens standard for å kalkulere kontrast og sette terskelverdier, utforskes nå i WCAG 3.0.
+
+<List.Root>
+  <List.Unordered>
+    <List.Item>
+      Metoden forbedrer hvordan verdien mellom to farger bestemmes, og gjør også
```

Comment:
Metoden forbedrer hvordan verdien mellom to farger bestemmes, og skiller også på om fargene er i forgrunnen eller i bakgrunnen.